packages/Link/Link.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story, Preview, Props, Subtitle } from '@storybook/addon-docs/blocks'
+import { withKnobs } from '@storybook/addon-knobs'
+import Link from './Link.jsx'
+import { version } from './package.json'
+
+<Meta title="Design System|Link" componenent={Link} />
+
+# Input
+<Subtitle>
+version {version}
+</Subtitle>
+
+## Usage
+* To install `Link` package with npm run terminal:
+
+```jsx
+npm i @pm-kit/input
+```
+* Import `Link` in your componenent to render
+
+```jsx
+import Link from '@pm-kit/link'
+```
+
+## Properties 
+
+<Props of={Link} />
+
+## Stories
+
+#### Default
+<Preview>
+<Story name="Default">
+  <Link link="http://www.google.com" target="_blank">Go To Google</Link>
+</Story>
+</Preview>
+
+#### Bold
+<Preview>
+<Story name="Bold">
+  <div style={{ fontWeight: 'bold' }}>
+    <Link link="http://www.google.com" target="_blank">Go To Google</Link>
+  </div>
+</Story>
+</Preview>
+
+#### Large
+<Preview>
+<Story name="Large">
+  <div style={{ fontSize: '32px' }}>
```

Comment:
let's use an example with the paragraph tag. It shows more cross-collaboration with other PM kit components. Also, let's show target="_blank" as an edge case, and not the default example.